ui_framework/doc_site/src/views/button/button_example.jsx

```diff
@@ -3,6 +3,8 @@ import React, {
   PropTypes,
 } from 'react';
 
+import { renderToHtml } from '../../services';
```

Comment:
This comment is meant for the file, but not sure how to add it at that level so adding here - looks like this is a `jsx` file... change to `js`?